core/package.json

```diff
@@ -70,6 +69,7 @@
     "sqlite3": "5.0.2",
     "ts-node": "10.0.0",
     "uuid": "8.3.2",
+    "umzug": "v3.0.0-beta.5",
```

Comment:
Unrelated, but umzug should be a dependency, as we want to allow users to use the `bin/migrator` command now